DigiMeSDK/Core/DMEMercuryInterfacer.h

```diff
@@ -0,0 +1,37 @@
+//
+//  DMEMercuryInterfacer.h
+//  DigiMeSDK
+//
+//  Created by Jacob King on 25/06/2018.
+//  Copyright © 2018 me.digi. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+typedef NSString DMEDigiMeOpenAction;
+
+@protocol DMEMercuryInterfacee;
```

Comment:
I understand the idea behind this implementation, but I think we should come up with a different name. Firstly we should not use internal project code names externally (Mercury). Secondly Interfacer/Interfacee doesn't sound intuitive :). Perhaps something along the lines of DMEAppClient + Handlers. We can discuss this.